src/styles/catalog.scss

```
@@ -0,0 +1,28 @@
+.catalog {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding-top: 50px;
+  padding-bottom: 72px;
+}
+
+.catalog__card {
```

pls make use of sass bem nesting
**PLS DO THE SAME FOR ALL SUCH CASES IN THE PROJECT**
```suggestion
  padding-bottom: 72px;
  
&__card {
```

---

Sure! Thank you for suggestion